versioned_docs/version-1.3.0-beta.1/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,172 @@
+The first step is to push the artifact with platform information to your container registry.
+
+First we push a Linux ARM64 executable:
+```console
+$ oras push --plain-http localhost:15000/oras:linux_arm64 --artifact-platform linux/arm64 --artifact-type 'application/vnd.oci.image.config.v1+json'  bin/linux/arm64/oras:application/octet-stream
+✓ Uploaded  bin/linux/arm64/oras                                                                                                     10.5/10.5 MB 100.00%   36ms
+  └─ sha256:32374a656f830f5d7504ad06c3c7be025fb38257d6793d8e70fed6f2b753c7ab
+✓ Exists    application/vnd.unknown.config.v1+json                                                                                       37/37  B 100.00%     0s
+  └─ sha256:d6f56bc20064075ce319ac2e6fcef5de9ea21773b0a8a4398c4405222971f9c0
+✓ Uploaded  application/vnd.oci.image.manifest.v1+json                                                                                 581/581  B 100.00%     0s
+  └─ sha256:302df4334cffe5132b6b81c06742be8ab3345bff3ce922662bc8674e79113bfd
+Pushed [registry] localhost:15000/oras:linux_amd64
+Tagged arm64
+ArtifactType: application/vnd.oci.image.config.v1+json
+Digest: sha256:302df4334cffe5132b6b81c06742be8ab3345bff3ce922662bc8674e79113bfd
+```
+
+Next, we push a Linux AMD64 executable:
+```console
+$ oras push --plain-http localhost:15000/oras:linux_amd64  --artifact-platform linux/amd64 --artifact-type 'application/vnd.oci.image.config.v1+json'  bin/linux/amd64/oras:application/octet-stream
+✓ Uploaded  bin/linux/amd64/oras                                                                                                         11/11 MB 100.00%     0s
+  └─ sha256:4640fefdb4e22eb1308e40d5fac9fd9d16e17dfbc20202bb3ed42033d01d823e
+✓ Exists    application/vnd.unknown.config.v1+json                                                                                       37/37  B 100.00%     0s
+  └─ sha256:9d99a75171aea000c711b34c0e5e3f28d3d537dd99d110eafbfbc2bd8e52c2bf
+✓ Uploaded  application/vnd.oci.image.manifest.v1+json                                                                                 581/581  B 100.00%     0s
+  └─ sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
+Pushed [registry] localhost:15000/oras:linux_amd64
+Tagged amd64
+ArtifactType: application/vnd.oci.image.config.v1+json
+Digest: sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
+```
+This example pushes the executable for two architectures, but you could also push artifacts with a different operating system or other platform attribute.
+
+## Create a multi-architecture manifest
+
+After you have pushed all your platform specific artifacts, create a manifest referencing them using the OCI image index format:
+```console
+$ oras manifest index create localhost:15000/oras:v1 sha256:302df4334cffe5132b6b81c06742be8ab3345bff3ce922662bc8674e79113bfd sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
```

Comment:
nit: How about putting the digests into environment variables for better readability?

Reply:
Done